main/src/ca/uwaterloo/flix/language/phase/Weeder.scala

```diff
@@ -357,7 +357,7 @@ object Weeder {
   /**
     * Performs weeding on the given enum case `c0`.
     */
-  private def visitCase(c0: ParsedAst.Case, enum: Name.Ident)(implicit flix: Flix): WeededAst.Case = c0 match {
+  private def visitCase(c0: ParsedAst.Case, ident: Name.Ident)(implicit flix: Flix): WeededAst.Case = c0 match {
```

Comment:
Shadows `ParsedAst.Case(...ident...) below, no?

Reply:
Dang, good catch.

Reply:
But it seems the name is not used?